<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GuiSelectItemsWidget</class>
 <widget class="QWidget" name="GuiSelectItemsWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>360</width>
    <height>387</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QLabel" name="mp_lSearch">
     <property name="minimumSize">
      <size>
       <width>24</width>
       <height>24</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>24</width>
       <height>24</height>
      </size>
     </property>
     <property name="pixmap">
      <pixmap resource="../beebeep.qrc">:/images/search.png</pixmap>
     </property>
     <property name="scaledContents">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="0" column="1" colspan="3">
    <widget class="QLineEdit" name="mp_leSearch">
     <property name="maxLength">
      <number>20</number>
     </property>
     <property name="placeholderText">
      <string>Search</string>
     </property>
    </widget>
   </item>
   <item row="0" column="4">
    <widget class="QPushButton" name="mp_pbClearFilter">
     <property name="maximumSize">
      <size>
       <width>48</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="toolTip">
      <string>Clear filter</string>
     </property>
     <property name="icon">
      <iconset resource="../beebeep.qrc">
       <normaloff>:/images/clear.png</normaloff>:/images/clear.png</iconset>
     </property>
    </widget>
   </item>
   <item row="1" column="0" colspan="5">
    <widget class="QTreeWidget" name="mp_tw">
     <column>
      <property name="text">
       <string notr="true">1</string>
      </property>
     </column>
    </widget>
   </item>
   <item row="2" column="0" colspan="2">
    <widget class="QPushButton" name="mp_pbClear">
     <property name="text">
      <string>Clear selection</string>
     </property>
     <property name="icon">
      <iconset resource="../beebeep.qrc">
       <normaloff>:/images/group-remove.png</normaloff>:/images/group-remove.png</iconset>
     </property>
    </widget>
   </item>
   <item row="2" column="2">
    <widget class="QPushButton" name="mp_pbOk">
     <property name="text">
      <string>Ok</string>
     </property>
    </widget>
   </item>
   <item row="2" column="3" colspan="2">
    <widget class="QPushButton" name="mp_pbCancel">
     <property name="text">
      <string>Cancel</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>mp_leSearch</tabstop>
  <tabstop>mp_pbClearFilter</tabstop>
  <tabstop>mp_tw</tabstop>
  <tabstop>mp_pbOk</tabstop>
  <tabstop>mp_pbClear</tabstop>
  <tabstop>mp_pbCancel</tabstop>
 </tabstops>
 <resources>
  <include location="../beebeep.qrc"/>
 </resources>
 <connections/>
</ui>
